CONTRIBUTING.md

```
@@ -69,13 +69,18 @@ All commands run from the repo root.
 | Test coverage | `pnpm --filter reka-ui test:coverage` |
 | Lint / auto-fix | `pnpm lint` / `pnpm lint:fix` |
 | Stories | `pnpm story:dev` |
-| Docs dev | `pnpm docs:install && pnpm docs:dev` |
+| Docs dev | `pnpm --filter reka-ui build && pnpm docs:install && pnpm docs:dev` |
 | Docs API regen | `pnpm docs:gen` (regenerates `docs/content/meta/*.md`) |
 | Bundle size | `pnpm --filter reka-ui size` |
 
 > `pnpm test` is **watch mode**. For a one-shot run (CI-style) use
 > `pnpm --filter reka-ui exec vitest run`.
 
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)